staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresource/etcd.go

```diff
@@ -91,7 +91,8 @@ func newREST(resource schema.GroupResource, listKind schema.GroupVersionKind, st
 		UpdateStrategy: strategy,
 		DeleteStrategy: strategy,
 
-		TableConvertor: tableConvertor,
+		TableConvertor:      tableConvertor,
+		ReturnDeletedObject: true,
```

Comment:
where is this used?

Reply:
In `testDeleteGracefulImmediate`, returned delete object is used to check the timestamps
```go
objectMeta = t.getObjectMetaOrFail(out)
	// the second delete shouldn't update the object, so the objectMeta.GetDeletionGracePeriodSeconds() should eqaul to the value set in the first delete.
	if objectMeta.GetDeletionTimestamp() == nil || objectMeta.GetDeletionGracePeriodSeconds() == nil || *objectMeta.GetDeletionGracePeriodSeconds() != 0 {
		t.Errorf("unexpected deleted meta: %#v", objectMeta)
	}
	if generation >= objectMeta.GetGeneration() {
		t.Error("Generation wasn't bumped when deletion timestamp was set")
	}
```

Reply:
I think I should remove here and skip the test. 
`CheckGracefulDelete` is used only in pod so I took inspiration from there